inference/core/interfaces/http/builder/editor.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <title>Workflow Editor</title>
+        <style>
+            html,
+            body {
+                height: 100%;
+                margin: 0;
+                padding: 0;
+                overflow: hidden;
+            }
+            iframe {
+                width: 100%;
+                height: 100%;
+                border: 0;
+            }
+        </style>
+        <script>
+            const BUILDER_ORIGIN = "{{BUILDER_ORIGIN}}";
+
+            document.addEventListener("DOMContentLoaded", function () {
+                // We'll parse the URL path. For example:
+                //   /build -> parts = ["", "build"]
+                //   /build/edit/foo -> parts = ["", "build", "edit", "foo"]
+                const parts = window.location.pathname
+                    .split("/")
+                    .filter(Boolean);
+
+                // Decide if it's /build/edit/:id or just /build
+                let workflowId = null;
+                if (parts.length >= 3 && parts[1] === "edit") {
+                    workflowId = parts[2];
+                }
+
+                // If we have an ID => /workflows/local/:id
+                // Else => /workflows/local
+                let iframeSrc = `${BUILDER_ORIGIN}/workflows/local`;
+                if (workflowId) {
+                    iframeSrc += `/${workflowId}`;
```

Comment:
this doesn't seem to work.  at first I though was missing part of the URL (missing `/edit/`, but that didnt fix it...so might be something about the react router on the app frontend